testing/daml/Testing/Init/Init.daml

```diff
@@ -52,15 +55,22 @@ data PvpContracts =
     delivery : ContractId SettlementInstruction
 
 
-allocateAndInitDemo : Script PartiesForTesting
+allocateAndInitDemo : Script UsersForTesting
 allocateAndInitDemo = script do
   -- Participants set up parties, connecting to the two Banking domains (EUR/GBP)
-  parties@PartiesForTesting{..} <- setupParties
+  users@EntitiesForTesting{..} <- setupPartiesAndUsers
+  let parties = usersToPartiesForTesting users
   initDemo parties
-  return parties
+  return users
+
+usersToPartiesForTesting : UsersForTesting -> PartiesForTesting
```

Comment:
Is it really needed to have top level functions for `usersToPartiesForTesting`, `userToParty`?

Reply:
addressed